main.js

```diff
@@ -254,6 +260,7 @@ function join() {
Aplv9lPyRM64OZztpT94rJJlO9NhbRAWNhAEaY+/RDZpaewBgF0txVgKQ++qSJFhruldbWTqyqr+NIZ6u9jT4IASPUv7QWTLZaXCtyTBLYdR19zb9BiCF/PWoEQEBkjjgBmtaeGg6EwPDtqT+s4n8+mTN01RjpKoAwCIE+axct3qKdTTTFlBC4Jk9wTwgcW/vgvleFPE2T5C+IAeOznzqkzaaQ1YFYPHcrbMkTEAghcOvgAC5CwGaTqH/1WZMjHkAYa+tvC0T7wSAliy9YqQqA+iqcFjTSF856MyRCQAyJF0UJ/9O6mMy+2DLYHOgQPx7UCIBoYAzppW9SVV68FQtMW7zmhMAfczZDoITAptIuVgCmhMB7xWJKCFSVdmiEQMs8LAiBN/KkzreQhBg2g7nqx6hMxBIovbu80ZS8LRch8OHgNLusCcGcF58ASwY0Yk/CCwCcOtK5QAyzrDAZj2lrXxstzO3mJU96A/2mB0JgUsfF+roRb92DsoLQDYZ/a+9rvQsRtvrVqBEAwZBZvAzynAXmftkHcw6MEHA6OCMhsKuEdTtdOLjACIHDvbUmBA7frc2wJS3yjiBsQqCIPQFQYCh2AQA6jwHYvosmhFvtu9xGBwD3hCDPW+as8z/vfzFXBQGFEgBGhWGE9uRJbaLpPizYqv7THvRaz2npgF10oO6Fp1E+0DQg3VQvx368e2i32bKRkaFGAARD+vAToXWXpHuR8wioxWvI0sCcTfLVZKOrMC1C2lHoB3K5lioWo5nN/bKttI3XX7qE8PQDgJYQmF4CTHbaRFOY4FMABwVFihefhPGWeSOb/NGB4lHDz5yp83ebaNqEWHc7goLMWGDxbkY0S4b6BHAkBKX2gMbWHSFwJk+l3RbRAUVB3frnjfmEjwwD/U4i9oOW/uaBchRAjxoBEAwZtQAQgK7tjKIjCJQ55w6aRwdU+Tq2eJnusDkhMEYHVPkGSAg0FIc0dbVJQvfkybClJgSuyRMcCYG28epNP6CAVb0AEAAoFiveX9K9W/rgtDZhGN0FxaOGn1YObt7zj+hP7gf9+/qam4LKTKFYkrHCtbu4XxbfrZEQ+CNPmhOYEIJE41ZII4FPqmeHCTDcITG6ISp1Py/KTgAUhtPLXQaRMHXGZ24E4WzNTp2DC4UQVIt+wHBYTQgC6i7gIQSm3QUm+WoSuiQE7rsLZtOLD/K9hKhfNA42ziw/7OhQv3t55ck3ATgrt6XxNPyEwK2D4yYE9g6OKdlBqX3bRxWGQCTqcYkmTc7kGYqEWan4nVqQTcd3kUyuvXBE/0R4yZPeQL+Z+27r1kck/mPYS8R+kaR5d/b3hTOTn1+UnQDs620RflayKMHmL1pl2Ong4EpHVa7UbSiNw8H5JgRR6Acswo5micuvHzDJV5NQfW8diBbgSAgI2IZ3MxHGPSlhf/dzGxbYqOdMCIH+NTbbckUIjCFt0r5gloRAYbEbu5reEHbJXimvATB+ZsGRJ7VJoPqBZAp1r4c69e8oMTwIwj0dvS0vABB6jeGyE4AaVLiVDgeyS4wH+Jiz7vOA5tMJgDnrDxqyVBMC1y1eMqkkCkdUFYC6YgkwOpDLtVSxGM1s7pdtmNJ4/dpLUN+v4KIDAAz6Acb2EwEgAJAU174n/JEqNSEwvmIWLV6DsbODsxAUZiDTrTaFLTskhWaV4jtOhkAfEAIh0HA2r4SAUL/xRbBk4EshTAJ4nBG7d97B9U82Njamgz5BWKgRAJEQk9/me61rjUlNP+CVEGgoDmluqUlC9+TJsJUnT56iA4AjISAApOwfAsAcqDSPsNtIlepdjFo/wIAHZz6+vMemcGWHTEhqZp40+W4DJwQB6Qfq1wcyGCIFYANjeFaR8dzKgWWvrcXaXGStP4jso0ONAIgEhb2NGEHTIgiIORc2QiEENf2AaanMNzkIgYo8BUoIJGm/EADmwBKW98vwLjpHU1wTAo/frcykX9sURghIUn4UgCkCIASO3611o8dOPyCNDCO+c5tFuS0xS0AjA9sGpmwHSVuy8eSrXV1dxS6mDrS4zVMY1AiASGD0tsJP7fvtgxBw6wdy+fIRAvcOzjch8KwfUOXrtmKxCelWlH4gdyOymcnRHdYJqgyEuHqomv5N19q6i6bk88//R8bX2GyL77ttTH74MKHXpgWAjIJknBVvLPiJPBl3laygvbfBREHVhKBu/TOAIgPAaO4h01j+RCME1s+IesCoh5HUrTClV5Koua2nrRkBTaAkImoEQCiwt2k+Fn39UbAKgTkXNkhla+pnAmrxGrJUEwJvDs7cTJUhc1uxlF0/YDTQPm5XlXATvrCyIteB9wIqLHlbfCyle6UhhaaJXRICzeMk42usti0amr1bdFMlzNIYyyJZXFDYRzRF/bEYebf/aIr+8MxHVp88cOO/7T8kmAM1AiAKbqZFIGVFaYd1CNzow4Nnzt6jAyZGDoTAPDqgytdVKyNva0IINNGB4nl4yRO4CYGG4pDmlpoktLtXgJ2gUNuwcowO7EfhfwBWXQCkew/+f3t3k5NADAUAuJ3BKCzceAXuYuJF8BQex8u48gocgIWixgXUDZqZMlNphISY71tOH00L74VH+attCCpysebzAyml1+lk+vhemPlcXF5sPj5DO1wyf2ievouleDqwC6tu5GN4Wt/OPflnNADn4ipMwibcj47nhdV0hnoDeTVUnl6N1uv+IWo7FN8OraH0TZiRl+57a+rENSEc41Quf7ugKcSGbuzPBNm+ur8+e8i+BuOyfeX3b2GRxTzYhrP739NTiik+pBiv+1cH8jBvCMIveZDnYnuc0+EmxeXq7ublKJOd2Go9f5vNlovU9q+3g9GH1sEutpfE28KDUTdvTOG54gYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADw330Bletj8G6CdnsAAAAASUVORK5CYII=`)
```

Comment:
This should be `server.favicon = (config.has("favicon") ? config.get("favicon") : null)`

Reply:
We would like a default favicon. So I disagree respectfully.